env/frontend.local.example.env

```diff
@@ -0,0 +1,2 @@
+# MITOPEN_AXIOS_WITH_CREDENTIALS=True
+# WEBPACK_ANALYZE=True
```

Comment:
`EMBEDLY_KEY=` here as well.

`MITOPEN_AXIOS_WITH_CREDENTIALS` isn't needed as it's in the committed file. Are the .local.env files intended only for secrets (aside from where it makes sense to group related variables)? Thinking `WEBPACK_ANALYZE` can go in `frontend.env` as well.